ocs_ci/deployment/acm.py

```diff
@@ -74,15 +74,27 @@ def __init__(self):
     def deploy(self):
         if self.source == "upstream":
             self.deploy_upstream()
-        else:
+        elif self.source == "downstream":
```

Comment:
I think that we should go with the downstream deployment by default without need of the passing extra configuration file, which we can achieve to set the default value in the .get method here: 
https://github.com/red-hat-storage/ocs-ci/blob/de15cfe75c67fd20ad711770f424d8888629f320/ocs_ci/deployment/acm.py#L63

Or setting it as a default value in the default configuration file here:
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/conf/default_config.yaml

@shylesh @prsurve @sidhant-agrawal @am-agrawa can you put your thoughts here? Is my assumption correct that by default we should go aways with downstream? And if from some reason we need upstream, we can always pass the config file for upstream?

Thanks

Reply:
yes, by default we should use downstream. In specific case, for eg. when using internal builds for ACM, we may need to use upstream.